### PR TITLE
Feature/message

### PR DESCRIPTION
**Pull request by peterdgreat**
- Created a table for storing your messages and create 5 different greetings.
- Created API endpoint that selects a random greeting from my table
- Created a static view that will be the root of your app.
- Created the App component with react-router and rendered it in my static view.
- Created the Greeting component that will display a greeting. 
- Created a store, an action and a reducer that will connect my API endpont to get the random greeting.
- Displayed the random greeting in your Greeting component

### CODE REVIEW

File: app/controllers/api/v1/greetings_controller.rb [hunk 1]
```diff
@@ -1,12 +1,8 @@
 class Api::V1::GreetingsController < ApplicationController
   def index
-       id = rand(1..Greeting.count)
+    id = rand(1..Greeting.count)
     @greeting = Greeting.find(id)
```
Comment (rloterh) [inline app/controllers/api/v1/greetings_controller.rb:4]: - [ ] Your app is displaying a blank screen, and hence no random greeting is being displayed. The seed file is properly run, and yet no data is displayed. Kindly check the attached screenshot of the console, an error pointing to the greetings controller, and kindly ensure to improve upon the same.

**Error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Peter, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/api/v1/greetings_controller.rb (hunk 1 above)
Comment (peterdgreat) [reply app/controllers/api/v1/greetings_controller.rb:4]: I populated it from my terminal, my bad.

**Comment by rloterh** [comment]
This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Peter, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**